Description: disable interactice parts of test2.p and test9.p
 which started to get exercised with giza-1.1.0
Origin: vendor
Bug-Debian: https://bugs.debian.org/914288
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2021-11-21

--- a/t/t2.t
+++ b/t/t2.t
@@ -60,19 +60,19 @@
 PGPLOT::pgptxt(25,2,35,0,'This way...');
 
 PGPLOT::pgqinf("CURSOR", my $ans, my $l);
-if ($ans eq "YES") {
-   diag "Entering interactive cursor test...\n";
-   PGPLOT::pgsci(4);
-   diag "Enter some points with the cursor, press 'x' when done\n";
-   our (@xt, @yt);
-   PGPLOT::pglcur(5, (my $n = 0),\@xt,\@yt);
-   PGPLOT::pgsci(9);
-   PGPLOT::pgpoint($n,*xt,*yt,20);
-   for(@xt) { diag sprintf "%5.2f ",$_; }
-   for(@yt) { diag sprintf "%5.2f ",$_; }
-   PGPLOT::pgsci(2);
-   PGPLOT::pgpoly($n,\@xt,\@yt);
-}
+#if ($ans eq "YES") {
+#   diag "Entering interactive cursor test...\n";
+#   PGPLOT::pgsci(4);
+#   diag "Enter some points with the cursor, press 'x' when done\n";
+#   our (@xt, @yt);
+#   PGPLOT::pglcur(5, (my $n = 0),\@xt,\@yt);
+#   PGPLOT::pgsci(9);
+#   PGPLOT::pgpoint($n,*xt,*yt,20);
+#   for(@xt) { diag sprintf "%5.2f ",$_; }
+#   for(@yt) { diag sprintf "%5.2f ",$_; }
+#   PGPLOT::pgsci(2);
+#   PGPLOT::pgpoly($n,\@xt,\@yt);
+#}
 
 PGPLOT::pgiden();
 
--- a/t/t9.t
+++ b/t/t9.t
@@ -66,19 +66,19 @@
 PGPLOT::pgptxt(12125,12100,45,0.5,'PGPLOT...');
 
 PGPLOT::pgqinf("CURSOR",my $ans,my $l);
-if ($ans eq "YES") { for(my $mode=0; $mode<8; $mode++){
-
-   note "Entering interactive PGBAND test MODE=$mode, hit any key, Q to exit early...";
-
-   PGPLOT::pgsci($mode+1);
-   PGPLOT::pgband($mode,0,12500,12500,my $x,my $y,my $ch);
-   last if $ch eq "q" || $ch eq "Q";
-   PGPLOT::pgqtxt($x,$y,45,0.5,'PGPLOT...',\@xbox,\@ybox);
-   PGPLOT::pgpoly(4,\@xbox, \@ybox);
-   PGPLOT::pgsci($mode+2);
-   PGPLOT::pgptxt($x,$y,45,0.5,'PGPLOT...');
-   
-}}
+#if ($ans eq "YES") { for(my $mode=0; $mode<8; $mode++){
+#
+#   note "Entering interactive PGBAND test MODE=$mode, hit any key, Q to exit early...";
+#
+#   PGPLOT::pgsci($mode+1);
+#   PGPLOT::pgband($mode,0,12500,12500,my $x,my $y,my $ch);
+#   last if $ch eq "q" || $ch eq "Q";
+#   PGPLOT::pgqtxt($x,$y,45,0.5,'PGPLOT...',\@xbox,\@ybox);
+#   PGPLOT::pgpoly(4,\@xbox, \@ybox);
+#   PGPLOT::pgsci($mode+2);
+#   PGPLOT::pgptxt($x,$y,45,0.5,'PGPLOT...');
+#
+#}}
 
 PGPLOT::pgend();
 
